src/main/scala/cromwell/engine/db/LiquibaseUtils.scala

```diff
@@ -0,0 +1,67 @@
+package cromwell.engine.db
+
+import java.sql.Connection
+
+import liquibase.database.DatabaseConnection
+import liquibase.database.jvm.{HsqlConnection, JdbcConnection}
+import liquibase.resource.{ClassLoaderResourceAccessor, ResourceAccessor}
+import liquibase.{Contexts, LabelExpression, Liquibase}
+import org.hsqldb.persist.HsqlDatabaseProperties
+
+import scala.collection.JavaConverters._
+
+object LiquibaseUtils {
+  val ChangeLogDir = "src/main/migrations/"
+  val DefaultChangeLog = "changelog.xml"
+  val DefaultResourceAccessor: ResourceAccessor = new ClassLoaderResourceAccessor()
+  val DefaultContexts = new Contexts()
+  val DefaultLabelExpression = new LabelExpression()
+
+  def updateSchema(jdbcConnection: Connection): Unit = {
+    val liquibaseConnection = newConnection(jdbcConnection)
+    try {
+      val liquibase = new Liquibase(DefaultChangeLog, DefaultResourceAccessor, liquibaseConnection)
+      checkForChangeLogDir(liquibase)
+      updateSchema(liquibase)
+    } finally {
+      closeConnection(liquibaseConnection)
+    }
+  }
+
+  def newConnection(jdbcConnection: Connection): DatabaseConnection = {
+    jdbcConnection.getMetaData.getDatabaseProductName match {
+      case HsqlDatabaseProperties.PRODUCT_NAME => new HsqlConnection(jdbcConnection)
+      case _ => new JdbcConnection(jdbcConnection)
+    }
+  }
+
+  def checkForChangeLogDir(liquibase: Liquibase): Unit = {
```

Comment:
Liquibase maintains its update log in the database, using paths to files or embedded resources. Before this PR, updates ran using files under the directory "src/main/migrations/". Now that we're using embedded resources instead of the files, the changelog must be updated to strip the partial or full file path from the change logs. I can add a comment to the method with an example if it makes this check clearer.